specifications/transactions/travel-rule/README.md

```diff
@@ -0,0 +1,372 @@
+
+## PaymentCommand
+
+All requests between participants are structured as `PaymentCommand`s within `CommandRequestObject`s:
+
+```
+{
+    "_ObjectType": "CommandRequestObject",
+    "command_type": "PaymentCommand",
+    "cid": "88b282d6-1811-29f6-82be-0421d0ee9887",
+    "command": {
+	    "_ObjectType": "PaymentCommand",
+	    "payment": {
+		    "sender": {
+			    "address": "dm1pg9q5zs2pg9q5zs2pg9q5zs2pg9skzctpv9skzcg9kmwta",
+			    "kyc_data": {
+				    "payload_version": 1,
+				    "type": "individual",
+				    "given_name": "ben",
+				    "surname": "maurer",
+				    "address": {
+					"city": "Sunnyvale",
+					"country": "US",
+					"line1": "1234 Maple Street",
+					"line2": "Apartment 123",
+					"postal_code": "12345",
+					"state": "California",
+				    },
+				    "dob": "1920-03-20",
+				    "place_of_birth": {
+					"city": "Sunnyvale",
+					"country": "US",
+					"postal_code": "12345",
+					"state": "California",
+				    }
+				},
+			    	"status": {
+			    		"status": "ready_for_settlement",
+			    	}
+			},
+		    "receiver": {
+			    "address": "dm1pgfpyysjzgfpyysjzgfpyysjzgf3xycnzvf3xycsm957ne",
+			},
+		    "reference_id": "5b8403c9-86f5-3fe0-7230-1fe950d030cb",
+		    "action": {
+			    "amount": 100,
+			    "currency": "USD",
+			    "action": "charge",
+			    "timestamp": 72322,
+			},
+		    "description": "A free form or structured description of the payment.",
+		},
+	},
+}
+```
+
+A successful response to this command should look like the following:
+
+```
+{
+    "_ObjectType": "CommandResponseObject",
+    "status": "success",
+}
+```
+
+A travel-rule exchange may require multiple `PaymentCommand`'s in order to complete. In these instances, the underlying structures may be passed back and forth many times. As outlined in the rest of this section, certain fields must be defined at the outset and others can be defined later. Some fields may be adjusted or mutated throughout the exchange, whereas others must remain immutable.
+
+### PaymentCommand Object
+
+| Field          | Type            | Required | Description |
+|-------         |------           |----------|-------------|
+| `_ObjectType`  | `str`           | Y        | The fixed string `PaymentCommand` |
+| `payment`      | `PaymentObject` | Y        | Contains a `PaymentObject` |
+
+```
+{
+    "_ObjectType": "PaymentCommand",
+    "payment": {
+        PaymentObject(),
+    }
+}
+```
+
+### PaymentObject
+
+Some fields are immutable after they are defined once. Others can be updated multiple times (see below). Updating immutable fields with a different value results in a Command error.
+
+| Field                           | Type                  | Required | Description |
+|-------                          |------                 |----------|-------------|
+| `sender`/`receiver`             | `PaymentActorObject`  | Y        | Information about the sender/receiver in this payment |
```

Comment:
"sender/receiver" should be 2 fields, merged into one line is like one field.

Reply:
hahaha, copy paste from the dip, but I agree and wondered if there was some reason for the single row